### PR TITLE
#159296016 Write unit tests for Registration.

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### What does this PR do?
These unit tests will check possible scenarios during the registration process.
#### How should this be manually tested?
$ python manage.py test --settings=authors.settings.testing
#### What are the relevant pivotal tracker stories?
#159296016

### CODE REVIEW

File: authors/apps/authentication/tests.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+
+        
+
+        
+
+
+
+
+
```
Comment (pluwum) [inline authors/apps/authentication/tests.py]: What are all these spaces for? 🤔

File: .travis.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ run:
 
 
 script:
-  - coverage run --source='.' manage.py test 
+  - coverage run --source='.' manage.py test --settings=authors.settings.testing
 
 after_success:
   - coveralls
```
Comment (pluwum) [inline .travis.yml]: Get rid of the red arrows when you see them

File: authors/apps/authentication/tests.py (hunk 1 above)
Comment (AGMETEOR) [reply authors/apps/authentication/tests.py]: I have removed them.

File: .travis.yml (hunk 2 above)
Comment (AGMETEOR) [reply .travis.yml]: I have removed them.

**Pull request merged**